### PR TITLE
Migration support for new columns

### PR DESCRIPTION
**Pull request by mattvb91**
Adding migration support for new mysql columns added in main repo:

https://github.com/phalcon/cphalcon/pull/10506

### CODE REVIEW

**Comment by sergeyklay** [comment]
Can you please add check for Phalcon version? Use new types if Phalcon >= 2.0.4

**Comment by mattvb91** [comment]
Of course. Will update this evening thanks.

File: ide/2.0.0/Phalcon/db/Column.php [hunk 1]
```diff
@@ -50,6 +50,31 @@ class Column implements \Phalcon\Db\ColumnInterface
...
+     * Mediumblob abstract data type
+     */
+    const TYPE_MEDIUMBLOB = 12;
+
+    /**
+     * Longblob abstract data type
+     */
+    const TYPE_LONGBLOB = 13;
+
+    /**
```
Comment (sergeyklay) [inline ide/2.0.0/Phalcon/db/Column.php]: And this should be in `ide/2.0.4/Phalcon/db/Column.php`

Comment (mattvb91) [inline ide/2.0.0/Phalcon/db/Column.php]: Thanks Sergey, Should ONLY the Phalcon/db/Column.php file be committed under the 2.0.4 folder or should I make a full copy of the 2.0.0 folder and commit it all?

Comment (sergeyklay) [inline ide/2.0.0/Phalcon/db/Column.php]: Support for new column types implemented in Phalcon 2.0.4.

If you need IDE subs for these types necessary to generate stubs from Phalcon `2.0.x` branch.

```sh
zephir stubs
```
Phalcon DevTools contains IDE stubs in `ide/<version>/Phalcon/` path

**Comment by mattvb91** [comment]
Sergey,

The generated files should they be comitted against cphalcon repo or copied into the devetools/ide folder? The zephir stubs generated class.zep.php files?

**Comment by sergeyklay** [comment]
1. Into devetools/ide folder like all other stubs.

2. Yes, it seems Zephir generates `*.zep.php` files. For example you can [rename the `*.zep.php` extension to `*.php` for a batch of files](http://stackoverflow.com/questions/1224766/how-do-i-rename-the-extension-for-a-batch-of-files). 

Or try to use [Stub Generator](https://github.com/phalcon/phalcon-devtools/blob/master/scripts/Phalcon/Generator/Stub.php)

**Comment by mattvb91** [comment]
The "stub generator" files were missing all the phpdoc comments on them? Have pushed the results using zephir stubs.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**